scripts/change-log-generator.js

```diff
@@ -119,8 +101,15 @@ function getNewChangeLogBranch(releaseBranch) {
     console.log('\nStep 2: Create new change log branch.');
   }
   var changeLogBranch =
-    CHANGE_LOG_BRANCH + releaseBranch.replace('origin/release/v', '');
-  shell.exec(util.format(GIT_CHECKOUT, changeLogBranch, releaseBranch));
+    constants.CHANGE_LOG_BRANCH +
+    releaseBranch.replace(constants.RELEASE_BRANCH_PREFIX, '');
+  shell.exec(`git checkout -b ${changeLogBranch} ${releaseBranch}`).code;
+  if (code !== 0) {
+    console.log(
+      'An error occurred generating the change log branch. Exitting.'
```

Comment:
Previously we would continue on with execution. This would mean the change log changes would be generated onto your current branch.

Reply:
nit: Exiting is misspelled.